canonical-web-and-design/ubuntu.com
# Add compliance takeover

## Done

- Build FIPS certification and CIS compliance takeover
- Add to `templates/takeovers/index.html`
- Add to `templates/index.html`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001
- Check takeover is visible on homepage and matches description on [issue](https://github.com/canonical-web-and-design/web-squad/issues/3209)
- Check CTA 

## Issue / Card

Fixes [#3209](https://github.com/canonical-web-and-design/web-squad/issues/3209)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/93783172-52b6fb80-fc23-11ea-81cd-5ad992a7fcba.png)
